src/Common/src/System/Security/Cryptography/Asn1/AlgorithmIdentifierAsn.xml

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<asn:Sequence
+  xmlns:asn="http://schemas.dot.net/asnxml/201808/"
+  name="AlgorithmIdentifierAsn"
+  namespace="System.Security.Cryptography.Asn1">
+    
+  <!--
+    https://tools.ietf.org/html/rfc3280#section-4.1.1.2
+
+    AlgorithmIdentifier ::= SEQUENCE {
+        algorithm OBJECT IDENTIFIER,
+        parameters ANY DEFINED BY algorithm OPTIONAL
+    }
+  -->
+  <asn:ObjectIdentifier name="Algorithm" />
```

Comment:
This breaks one unit test because it populates the friendly name. I am aware of that and I will fix it. Moreover, the whole class is actually duplicated in S.S.C.X509Certificates as `X501AttributeAsn`, where `Parameters` is declared as array. I will clean that up in a separate PR along with the remaining usages of serializer, resources and project files.

Reply:
Fixed the unit test failure, marking as resolved.